hyperspy/misc/hyperspy_magics.py

```diff
@@ -62,29 +69,47 @@ def hyperspy(self, line):
         else:
             toolkit = preferences.General.default_toolkit
 
-        if toolkit not in ["None", "none"]:
+        if toolkit not in ['qt4', 'gtk', 'wx', 'tk', 'None', 'none']:
+            raise ValueError("The '%s' toolkit is not supported.\n" % toolkit +
+                             "Supported toolkits: {qt4, gtk, wx, tk, None}")
+
+        mpl_code = ""
+        if toolkit in ["None", "none"]:
```

Comment:
True. I'll change that, as I said.

I wrote it this way at first because that was the only option that was not passed to the `%matplotlib` magic, hence was not handled by matplotlib (however gracefully)